sealights.gradle

```diff
@@ -1,14 +1,3 @@
-buildscript {
```

Comment:
just to make sure, will sealights plugin be working without that section?

Reply:
No